src/index.html

```diff
@@ -3,11 +3,318 @@
+          <p class="details__text-bottom">
+            The app senses when you're nearby to unlock automatically.
+            GPS tracking so you know where your bike is and can track it anytime.
+          </p>
+        </div>
+
+        <div class="details__container">
+          <div class="details__image">
+            <img src="./image/integrated1.png"
+            alt="bike-left"
+            class="details__image-left"
+            >
+
+            <img src="./image/integrated2.png"
+            alt="bike-right"
+            class="details__image-right"
+            >
+          </div>
+        </div>
+        <div class="details__text">
+          <h3 class="details__title-bottom">Range & Integrated lights</h3>
+          <p class="details__text-bottom">
+             The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+            Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </div>
+
+        <div class="details__container">
+          <div class="details__image">
+            <img src="./image/hydraulic1.png"
+            alt="bike-left"
+            class="details__image-left details__image-left-bottom"
+            >
+
+            <img src="./image/hydraulic2.png"
+            alt="bike-right"
+            class="details__image-right details__image-right-bottom"
+            >
+          </div>
+        </div>
+        <div class="details__text">
+          <h3 class="details__title-bottom">Hydraulic disc brakes & Lightweight</h3>
+          <p class="details__text-bottom">
+            Brakes with total stopping power the second you make contact.
+            The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+            Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </div>
+
+        <button
+          class="button__details"
+          type="submit"
+          >
+          Explore
+        </button>
+      </section>
+
+      <section class="contacts" id="contacts">
+        <h2 class="contacts__title">Contact us</h2>
+
+        <div class="contacts__container">
+          <div class="contacts__form">
+            <form
+            onsubmit="this.reset(); return false;"
+            class="form"
+            >
+
+              <input
+              type="text"
+              name="text"
+              placeholder="Name"
+              required
+              class="form__input"
+              >
+
+              <input
+              type="email"
+              name="email"
+              placeholder="Email"
+              required
+              class="form__input"
+              >
+
+              <textarea
+              type="text"
+              name="message"
+              placeholder="Message"
+              required
+              class="form__input textarea"
+              >
+              </textarea>
```

Comment:
```suggestion
              <textarea
              type="text"
              name="message"
              placeholder="Message"
              required
              class="form__input textarea"
              ></textarea>
```

Update format to fix extra spaces in textarea field